scripts/make-g-cloud-7-live.py

```diff
@@ -0,0 +1,120 @@
+"""
+
+Usage:
+    scripts/make-g-cloud-7-live.py <data_api_url> <data_api_token> <draft_bucket> <documents_bucket> [--dry-run]
+"""
+import sys
+sys.path.insert(0, '.')
+
+from itertools import islice
+import random
+import re
+
+from docopt import docopt
+from dmutils.apiclient import DataAPIClient
+from dmutils.s3 import S3
+
+DOCUMENT_KEYS = [
+    'pricingDocumentURL', 'serviceDefinitionDocumentURL',
+    'sfiaRateDocumentURL', 'termsAndConditionsDocumentURL',
+]
+FRAMEWORK_SLUG = 'g-cloud-7'
+
+
+def assert_equal(one, two):
+    if one != two:
+        raise AssertionError("{} != {}".format(one, two))
+
+
+def find_suppliers_on_framework(client, framework_slug):
+    return (
+        supplier for supplier in client.find_framework_suppliers(FRAMEWORK_SLUG)['supplierFrameworks']
+        if supplier['onFramework']
+    )
+
+
+def find_submitted_draft_services(client, supplier, framework_slug):
+    return (
+        draft_service for draft_service in
+        client.find_draft_services(supplier['supplierId'], framework=framework_slug)['services']
+        if draft_service['status'] == 'submitted' and not supplier.get('serviceId')
```

Comment:
Should this be `and not service.get('serviceId')` as suppliers don't have service IDs?